sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseDataDto.java

```diff
@@ -1720,8 +1724,13 @@ public void setHealthConditions(HealthConditionsDto healthConditions) {
 		this.healthConditions = healthConditions;
 	}
 
+	@JsonIgnore
+	public String getI18nPrefix() {
+		return I18N_PREFIX;
+	}
+
 	@Override
 	public String toString() {
-		return this.getUuid() + " - " + this.getExternalID();
+		return super.toString() + (StringUtils.isNotBlank(this.getExternalID()) ? " - " + this.getExternalID() : StringUtils.EMPTY);
```

Comment:
Why is the `externalID` part of the toString() method. Should only be standard imho.

Reply:
was like that before
maybe @MateStrysewske can weigh in on if it is really needed

Reply:
Pretty sure it's important for Germany which is why it should stay there, but only if it's filled in (so, as it is right now)